src/components/ButtonGroup.js

```diff
@@ -0,0 +1,95 @@
+const React = require('react');
+const Radium = require('radium');
+
+const Button = require('./Button');
+
+const StyleConstants = require('../constants/Style');
+
+const ButtonGroup = React.createClass({
+  propTypes: {
+    buttons: React.PropTypes.arrayOf(React.PropTypes.shape({
+      icon: React.PropTypes.string,
+      text: React.PropTypes.string
+    }).isRequired),
+    primaryColor: React.PropTypes.string,
+    type: React.PropTypes.oneOf([
+      'base',
+      'disabled',
+      'neutral',
+      'primary',
+      'primaryOutline'
+    ])
+  },
+
+  getDefaultProps () {
+    return {
+      buttons: [],
+      primaryColor: StyleConstants.Colors.PRIMARY,
+      type: 'primaryOutline'
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+    const buttonType = this.props.type;
+    const primaryColor = this.props.primaryColor;
+    const buttons = this.props.buttons;
+
+    return (
+      <div {...this.props}>
+        {this.props.buttons.map(function (button, i) {
+          const isFirstChild = i === 0;
+          const isLastChild = i === buttons.length - 1;
+          const isOnlyChild = isFirstChild && isLastChild;
+
+          return (
+            <Button
+              icon={button.icon}
+              key={i}
+              primaryColor={primaryColor}
+              style={[
+                styles.component,
+                styles[buttonType],
```

Comment:
I may be missing something, but I am not seeing these styles defined. Should there be a styles.primaryOutline there or is that handled in the Button component?